src/test/three-components/ModelScene-spec.js

```diff
@@ -26,13 +26,6 @@ function invertPad(vec3) {
   return vec3.clone().multiplyScalar(ROOM_PADDING_SCALE);
 }
 
-// Checks that the skysphere is within the camera's far plane
-function ensureSkysphereVisible(scene) {
```

Comment:
Testing and assertions reduced here. What changed?

Reply:
The positioning of the skysphere is in the hands of three now, and the environment feature tests cover whether or not it's visible, these tests here ensured the skybox was positioned and sized correctly

Reply:
👍